src/prop_types.js

```diff
@@ -82,6 +82,21 @@ export let VLayoutItemPropTypes = {
 
   // Used internally by VLayout
   _gutterTop: React.PropTypes.string,
-  _gutterBottom: React.PropTypes.string
+  _gutterBottom: React.PropTypes.string,
+
+  style: function (props, propName, componentName) {
+    const DANGEROUS_STYLES = [
+      "padding", "paddingTop", "paddingBottom", "paddingLeft", "paddingRight",
+      "margin", "marginTop", "marginBottom", "marginLeft", "marginRight",
+      "height", "width", "display", "position", "float",
```

Comment:
It can, but I'd lean towards making a user do this through the `width` prop rather than allowing them to do it through styles. It seems odd to have two different ways of controlling the same thing on a component

I'll add it to the HLayoutItem and open up another pull request